docs/readthedocs_theme/base.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   {% block htmltitle %}
   <title>{{ page_title }}</title>
+  {% if page_description %}<meta name="description" content="{{ page_description }}">{% endif %}
```

Comment:
Just checked the currently available schema with my wife (who is doing SEO stuff) and it seems that the dialects are mostly pointless anyways as the titles are too long for google. 

Example:

```Working with legacy tables - Sequelize - node.js ORM for Postgres, MySQl, MariaDB and SQLite```

-->

Peeps will see / Google will rate only:

![serp](https://s3.amazonaws.com/f.cl.ly/items/1g0F00432V0w413V001k/Image%202015-02-15%20at%2010.24.25%20nachm..png)

Having that said I'd vote for the shorter title.